Updated meta descriptions flagged as 'too short' (#724)
MicrosoftDocs/quantum-docs-pr: articles/concepts/vectors-and-matrices.md

---
title: Vectors and matrices in quantum computing

> description: Learn the basics of vector and matrix multiplication and notation as it relates to quantum computing.

Reviewer: ```suggestion
description: Learn the basics of how to work with vectors and matrices.
```
